Include/object.h

```diff
@@ -581,13 +581,13 @@ PyAPI_FUNC(PyObject*) Py_NewRef(PyObject *obj);
 // Similar to Py_NewRef(), but the object can be NULL.
 PyAPI_FUNC(PyObject*) Py_XNewRef(PyObject *obj);
 
-static inline PyObject* _Py_NewRef(PyObject *obj)
+static inline PyObject* Py_ALWAYS_INLINE _Py_NewRef(PyObject *obj)
```

Comment:
On Windows, this gives syntax error because ``__forceinline`` specifier should come before the return type. This applies to all the other errors for GHA below.
```suggestion
static inline Py_ALWAYS_INLINE  PyObject* _Py_NewRef(PyObject *obj)
```
However, I don't know if this will work for GCC/Clang. Also I noticed that mpdecimal module has its own ALWAYS_INLINE https://github.com/python/cpython/blob/bb3e0c240bc60fe08d332ff5955d54197f79751c/Modules/_decimal/libmpdec/mpdecimal.c#L65.

Reply:
Clang has no problems with the specifier placement; it can be either before or after. No problem :)

Reply:
Oh, good to know: fixed. I explained it in the macro documentation and I fixed the usage in the doc.